carbon/carbon_informatie_4.pv

```diff
@@ -15,121 +15,49 @@ print years
 # data (van lage naar hoge waardes) 
 
 # agriculture
-data = gasData[0] 
-print data['sector']
+agri_data = gasData[0] 
+print agri_data['sector']
 
 # built-up area
-data = gasData[3]
-print data['sector']
+build_data = gasData[3]
+print build_data['sector']
 
 # traffic and transport
-data = gasData[2]
-print data['sector']
+traffic_data = gasData[2]
+print traffic_data['sector']
 
 # industry and energy 
-data = gasData[1]
-print data['sector'] 
+industry_data = gasData[1]
+print industry_data['sector'] 
 
 
 # forloop 
 
-for y in years:
-	print data[y]
-
 x = 30	
 for year in years:
     x = x + 16
-    barheight = data[year] * multiply
-    barheightagriculture= data[0][year] * multiply
-    barheightbuiltup= data[3][year] * multiply
-    barheighttraffic= data[2][year] * multiply
-    barheightindustry= data[1][year] * multiply
+    # barheight = data[year] * multiply
+    barheightagriculture = agri_data[year] * multiply
+    barheightbuiltup= build_data[year] * multiply
+    barheighttraffic= traffic_data[year] * multiply
+    barheightindustry= industry_data[year] * multiply
 
 
     fill('#0000ff')
-    rect(x, 500 - barheight ,12, barheight)
+    from_y = 500 - barheightindustry
+    rect(x, from_y ,12 , barheightindustry)
     
     fill('#800080')
-    rect(x, 500 - barheight-barheightagriculture ,12, barheight-barheightagriculture)
+    from_y = 500 - (barheightindustry + barheightagriculture)
+    rect(x, from_y, 12, barheightagriculture)
     
     fill ('#000')
-    rect(x, 500 - barheight-barheightagriculture-barheightbuiltup ,12, barheight-barheightagriculture-barheightbuiltup) 
+    from_y = 500 - (barheightindustry + barheightagriculture + barheightbuiltup)
```

Comment:
Use the `500 - ( a + b + c)` construction instead of `500 - a - b -c`, again to make it more readable and less multi-interpretable.